cvs diff: Diffing sfx2
cvs diff: Diffing sfx2/inc
cvs diff: Diffing sfx2/inc/pch
cvs diff: Diffing sfx2/inc/sfx2
cvs diff: Diffing sfx2/prj
cvs diff: Diffing sfx2/qa
cvs diff: Diffing sfx2/qa/complex
cvs diff: Diffing sfx2/qa/complex/DocHelper
cvs diff: Diffing sfx2/qa/complex/docinfo
cvs diff: Diffing sfx2/qa/unoapi
cvs diff: Diffing sfx2/qa/unoapi/testdocuments
cvs diff: Diffing sfx2/sdi
cvs diff: Diffing sfx2/source
cvs diff: Diffing sfx2/source/appl
cvs diff: Diffing sfx2/source/bastyp
cvs diff: Diffing sfx2/source/config
cvs diff: Diffing sfx2/source/control
cvs diff: Diffing sfx2/source/dialog
cvs diff: Diffing sfx2/source/doc
cvs diff: Diffing sfx2/source/explorer
cvs diff: Diffing sfx2/source/inc
cvs diff: Diffing sfx2/source/inet
cvs diff: Diffing sfx2/source/menu
cvs diff: Diffing sfx2/source/notify
cvs diff: Diffing sfx2/source/statbar
cvs diff: Diffing sfx2/source/toolbox
cvs diff: Diffing sfx2/source/view
Index: sfx2/source/view/viewsh.cxx
===================================================================
RCS file: /cvs/framework/sfx2/source/view/viewsh.cxx,v
retrieving revision 1.79
retrieving revision 1.79.44.1
diff -u -p -b -w -B -r1.79 -r1.79.44.1
--- sfx2/source/view/viewsh.cxx	7 Nov 2007 10:44:11 -0000	1.79
+++ sfx2/source/view/viewsh.cxx	8 Jan 2008 14:30:42 -0000	1.79.44.1
@@ -912,18 +912,9 @@ void SfxViewShell::Activate( BOOL bMDI )
 
 //--------------------------------------------------------------------
 
-void SfxViewShell::Deactivate(BOOL bMDI)
+void SfxViewShell::Deactivate(BOOL /*bMDI*/)
 {
 	DBG_CHKTHIS(SfxViewShell, 0);
-	if ( bMDI )
-	{
-		SfxObjectShell *pSh = GetViewFrame()->GetObjectShell();
-        Reference< XModel > xDoc;
-        if ( pSh )
-            xDoc = pSh->GetModel();
-		if ( xDoc.is() && ( xDoc == SfxObjectShell::GetWorkingDocument() ) )
-		    SfxObjectShell::SetWorkingDocument( Reference< XModel >() );
-    }
 }
 
 //--------------------------------------------------------------------
cvs diff: Diffing sfx2/util
cvs diff: Diffing sfx2/util/defs
cvs diff: Diffing sfx2/win
cvs diff: Diffing sfx2/win/res
